packages/bpk-component-nudger/README.md

```diff
@@ -47,19 +49,104 @@ class App extends Component {
 }
 ```
 
+### Configurable `BpkConfigurableNudger`
+
+```js
+import React, { Component } from 'react';
+import BpkLabel from 'bpk-component-label';
+import { BpkConfigurableNudger } from 'bpk-component-nudger';
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      value: 1,
+    };
+  }
+
+  handleChange = value => {
+    this.setState({ value });
+  };
+
+  compareValues = (a, b): number => {
+    const options = ['economy', 'premium', 'business', 'first'];
+    const [aIndex, bIndex] = [options.indexOf(a), options.indexOf(b)];
+    return aIndex - bIndex;
+  };
+
+  incrementValue = (a): string => {
+    const options = ['economy', 'premium', 'business', 'first'];
+    const [aIndex] = [options.indexOf(a) + 1];
+    return options[aIndex];
+  };
+
+  decrementValue = (a): string => {
+    const options = ['economy', 'premium', 'business', 'first'];
+    const [aIndex] = [options.indexOf(a) - 1];
+    return options[aIndex];
+  };
+
+  formatValue = (a): string => a.toString();
+
+  render() {
+    return (
+      <div>
+        <BpkLabel htmlFor="my-nudger">Number of passengers</BpkLabel>
+        <BpkConfigurableNudger
+          id="nudger"
+          min="economy"
+          max="first"
+          value={this.state.value}
+          onChange={this.handleChange}
+          decreaseButtonLabel="Decrease"
+          increaseButtonLabel="Increase"
+          compareValues={this.compareValues}
+          incrementValue={this.incrementValue}
+          decrementValue={this.decrementValue}
+          formatValue={this.formatValue}
+          inputClassName={getClassName('bpk-nudger-configurable')}
```

Comment:
I'd just remove this line, `getClassName` isn't defined and we generally only document the most obvious props and not the more 'advanced' stuff like passing in class names.